src/Aquifer.Public.API/Endpoints/Resources/Updates/List/Endpoint.cs

```diff
@@ -97,4 +98,13 @@ private async Task<IQueryable<ResourceContentVersionEntity>> GetQuery(Request re
                         (req.ResourceCollectionCode == null ||
                          x.ResourceContent.Resource.ParentResource.Code.ToLower() == req.ResourceCollectionCode.ToLower()));
     }
+
+    private void ValidateCollectionCode(Request req)
+    {
+        var exists = dbContext.ParentResources.Any(x => x.Code == req.ResourceCollectionCode);
```

Comment:
Resource collection code is nullable. I would think we would verify it's not null before calling the database.

Reply:
good idea